source/Government.cpp

```diff
@@ -510,3 +515,10 @@ bool Government::IsProvokedOnScan() const
 {
 	return provokedOnScan;
 }
+
+
+
+bool Government::IsUsingForeignPenaltiesFor(const Government *government) const
```

Comment:
This function isn't really needed, so I'd remove it.

Reply:
What do you mean? Its used in politics